packages/repository/src/decorators/model.ts

```diff
@@ -15,7 +15,7 @@ export const PROPERTY_KEY = 'loopback:property';
  * @returns {(target:AnyType)}
  */
 export function model(definition?: Object) {
-  return function(target: AnyType) {
+  return function(target: Any) {
```

Comment:
The `Reflect` API expects `target` to be of `Object` type, see e.g. https://github.com/rbuckton/reflect-metadata/blob/d20867eca6503452be795f10b2fb6eb7e713dc25/Reflect.d.ts#L139. I think we should preserve that constraint in our API too.